plugins/defaultplugins/aclplugin/vppdump/dump_vppcalls.go

```diff
@@ -97,22 +96,19 @@ func DumpIPAcl(log logging.Logger, vppChannel *govppapi.Channel, timeLog measure
 		if stop {
 			break
 		}
-		log.Infof("ACL index: %v, rule count: %v, tag: %v", msg.ACLIndex, msg.Count, string(msg.Tag[:]))
+		acli = append(acli, msg.ACLIndex)
 
-		accessList := &acl.AccessLists_Acl{}
-		// Rules
-		accessList.Rules = getMatchRules(msg.R)
-
-		// Interfaces
```

Comment:
the logic has changed - you need to first get the ACL indices, and then the details for each index. You can't interleave getting an index with getting the details, because of the context.  But you are right about the interfaces - i did not have time to implement it yet, that will be comin in a subsequent patch. But it will be a part of the getIPACLDetails() function.

Reply:
I was working on the same, so to prevent any merge conflicts I will abandon mine. It is also the bug with wrong message ID when dumping ACLs